backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -49,19 +52,35 @@ async def jobs(
             status_code=422,
         )
 
-    results = await getData(start_date, end_date, "ocp.elasticsearch")
+    if not offset:
+        offset = 0
 
-    if len(results) >= 1:
+    if not size:
+        size = 10000
+        offset = 0
+
+    if not sort:
+        sort = None
```

Comment:
Is this just to convert an explicit `?sort=''` from an empty string into `None`? Because I think that's all it's doing as the default is already `None`.

Reply:
What if there is no `sort` param in the request payload